frontend/src/Components/Charts/DumbbellChart/DumbbellChart.tsx

```diff
@@ -63,19 +63,21 @@ const sortDataHelper = (originalData: DumbbellDataPoint[], sortModeInput: 'preop
 };
 
 type Props = {
-    data: DumbbellDataPoint[];
-    xAxisVar: DumbbellLayoutElement['xAxisVar'];
-    dimensionWidth: number,
-    dimensionHeight: number;
-    svg: React.RefObject<SVGSVGElement>;
-    xMin: number;
-    xMax: number;
-    sortMode: 'preop' | 'postop' | 'gap';
-    showPreop: boolean;
-    showPostop: boolean;
+  data: DumbbellDataPoint[];
+  xAxisVar: DumbbellLayoutElement['xAxisVar'];
+  dimensionWidth: number,
+  dimensionHeight: number;
+  svg: React.RefObject<SVGSVGElement>;
+  xMin: number;
+  xMax: number;
+  sortMode: 'preop' | 'postop' | 'gap';
+  showPreop: boolean;
+  showPostop: boolean;
+  hgbPostOpTargetRange?: (number | undefined)[];
+  hgbPreOpTargetRange?: (number | undefined)[];
```

Comment:
These are always defined, right?
```suggestion
  hgbPostOpTargetRange: (number | undefined)[];
  hgbPreOpTargetRange: (number | undefined)[];
```